src/meta/model_v2/migration/src/lib.rs

```diff
@@ -24,6 +25,7 @@ impl MigratorTrait for Migrator {
             Box::new(m20240410_154406_session_params::Migration),
             Box::new(m20240417_062305_subscription_internal_table_name::Migration),
             Box::new(m20240418_142249_function_runtime::Migration),
+            Box::new(m20240422_090457_secret::Migration),
```

Comment:
I'm not sure whether a smaller date naming for this migration can be applied when upgrade from the version before this PR. Didn't check, if not work you can rename to a larger date for it.

Reply:
fixed